.travis.yml

```diff
@@ -12,7 +12,7 @@ env:
         - deps=no
 
 before_script:
-    - if [ "$deps" = "low" ]; then composer --prefer-lowest --prefer-stable update; fi;
```

Comment:
well, removing ``--prefer-lowest`` is wrong. It means that deps=low does not test low deps anymore.

If tests are broken using the lowest deps, they should be fixed, not bypassed